components/bl-audio-recorder-component/src/index.js

```diff
@@ -32,8 +35,14 @@ export default function AudioRecorder({ component, eventHandlers, elRef }) {
     if (state) {
       onStateChange({ state });
     }
+
```

Comment:
You pass state and StreamState.RECORDING(constant) to compare them inside the function - for me, it looks incorrect. Also as a storing everything in state. Do we need to rerender when timerInreval is set? I cant see anything that relies on it.